src/main/app/src/app/zoeken/zoek/zoek.component.html

```diff
@@ -109,7 +109,7 @@
         matSuffix
         id="wissen-button"
       >
-        <mat-icon outlined>close</mat-icon>
+        <mat-icon>close</mat-icon>
```

Comment:
Does this change the layout of the button? There must've been a reason for the `outlined` icon.